src/WPF/ArcGISRuntime.WPF.Viewer/ArcGISRuntime.WPF.Viewer.NetFramework.csproj

```diff
@@ -148,6 +148,9 @@
   </ItemGroup>
```

Comment:
There are several issues with this file, I will open a separate PR to update it.